patches/chrome-browser-push_messaging-push_messaging_service_impl.cc.patch

```diff
@@ -1,14 +0,0 @@
-diff --git a/chrome/browser/push_messaging/push_messaging_service_impl.cc b/chrome/browser/push_messaging/push_messaging_service_impl.cc
-index ce505a8d3d58d0c1b0c96e6b3f6280f0e26576ba..85f28d93c86f925be869d9ec8944ae5e46674490 100644
---- a/chrome/browser/push_messaging/push_messaging_service_impl.cc
-+++ b/chrome/browser/push_messaging/push_messaging_service_impl.cc
-@@ -169,7 +169,8 @@ void PushMessagingServiceImpl::InitializeForProfile(Profile* profile) {
- 
-   PushMessagingServiceImpl* push_service =
-       PushMessagingServiceFactory::GetForProfile(profile);
--  push_service->IncreasePushSubscriptionCount(count, false /* is_pending */);
-+  if (push_service)
```

Comment:
this is now in upstream?

Reply:
Yes! https://chromium.googlesource.com/chromium/src/+/8fe6c81394a006846510d4dd4b62ba6f99681b54%5E%21/#F0